st2tests/st2tests/fixtures/packs/dummy_pack_22/config.schema.yaml

```diff
@@ -0,0 +1,8 @@
+---
+  wrong_key_with_invalid_default_value:
+    type: "string"
+    secret: true
+    required: true
+    default: "{{st2kv.user.api_key | decrypt_kv}}"
+  key_with_no_secret_and_no_default:
+    type: "string"
```

Comment:
Please get rid of the end of line character.

Reply:
Done